superbit_lensing/shear_profiles/annular_jmac.py

```diff
@@ -193,27 +270,38 @@ def main(args):
     #y_arg = 'y_image'
     #startrad = 180
     #endrad = 4000
-    nfw_center = [5031,3353]
+    nfw_center = [5031, 3353]
```

Comment:
We will probably want to make this configurable at some point.

Reply:
agreed - so this specific bit never gets called anymore though, as the `main()` func now exists only for legacy calls. `make_annular_catalog.py` builds an `Annular` object explicitly now and sets it's configuration, so we would generalize there

Reply:
I see that now, thanks. I'll add making the NFW center configurable to my wishlist of codebase enhancements.